models/branches.go

```diff
@@ -35,6 +35,8 @@ type ProtectedBranch struct {
 	EnableMergeWhitelist      bool           `xorm:"NOT NULL DEFAULT false"`
 	MergeWhitelistUserIDs     []int64        `xorm:"JSON TEXT"`
 	MergeWhitelistTeamIDs     []int64        `xorm:"JSON TEXT"`
+	EnableStatusCheck         bool           `xorm:"NOT NULL DEFAULT false"`
```

Comment:
Can you make a migration to add these two columns?

Reply:
done.